microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -22,145 +22,166 @@ ms.date: 05/04/2021
    limitations under the License.  -->
```

Comment:
Line 18, Column 3: [Warning: bookmark-not-found] Cannot find bookmark '
#step-4-save-a-javascript-change-to-disk
in 'devtools-guide-chromium/workspaces/index.md'.
Change to:
#edit-javascript-and-save-changes-to-the-source-file

obtained from a commit-specific rendered view:
https://github.com/MicrosoftDocs/edge-developer/blob/4f63ef9569387f34c5d8aae8f74c066abc74d067/microsoft-edge/devtools-guide-chromium/workspaces/index.md#edit-javascript-and-save-changes-to-the-source-file

Reply:
made fix